src/runtime_src/core/edge/drm/zocl/scu.c

```diff
@@ -264,3 +247,20 @@ int zocl_scu_wait_cmd_sk(struct platform_device *pdev)
 
 	return 0;
 }
+int zocl_scu_sk_ready(struct platform_device *pdev)
```

Comment:
Can the return type for this function be a void instead of an int for it always returns zero.